Libraries/Text/TextInput/RCTBackedTextInputDelegateAdapter.m

```diff
@@ -191,11 +203,17 @@ - (BOOL)control:(NSControl *)control textView:(NSTextView *)fieldEditor doComman
   BOOL commandHandled = NO;
   // enter/return
   if (commandSelector == @selector(insertNewline:) || commandSelector == @selector(insertNewlineIgnoringFieldEditor:)) {
-    [self textFieldDidEndEditingOnExit];
-    if ([textInputDelegate textInputShouldReturn]) {
-      [[_backedTextInputView window] makeFirstResponder:nil];
+  #if TARGET_OS_OSX // [TODO(macOS Candidate GH#774)
```

Comment:
Don't need "Candidate" (and would also make it easier for me to swap these out if  you remove the word)